src/renderer/components/SplittedChatListAndView.js

```diff
@@ -41,6 +41,7 @@ class SplittedChatListAndView extends React.Component {
     this.onCreateContact = this.onCreateContact.bind(this)
     this.initiateKeyTransfer = this.initiateKeyTransfer.bind(this)
     this.onKeyTransferComplete = this.onKeyTransferComplete.bind(this)
+    this.markNoticedChatIfNeeded = this.markNoticedChatIfNeeded.bind(this)
```

Comment:
this is only needed when the function is used as a button in a React element, so you can remove it